.changeset/spicy-eggs-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vocab/phrase': patch
+---
+
+Upload translations in a separate file per language
```

Comment:
Should this include a reference to what it fixes?

Reply:
Oops, forgot to update this from when I was testing snapshots. Will update.

Reply:
Any opposition to more-or-less copying and pasting the PR description into the changeset?

Reply:
Added some detail to the changeset https://github.com/seek-oss/vocab/pull/152/commits/c2887138aa4d0433d79deb124741147c1393b5bc

Reply:
Looks good